src/main/scala/nl.knaw.dans.easy.dd2d/mapping/Description.scala

```diff
@@ -42,4 +42,11 @@ object Description extends BlockCitation {
     m.addPrimitiveField(DESCRIPTION_VALUE, s"$prefix: ${ node.text }")
     m.toJsonObject
   }
+
+  def newlineToHtml(description: String): String = {
+    description.split("\n{2}")
```

Comment:
* You will get an empty paragraph if the description starts with two new lines.
* Can the description somehow originate from windows and thus also have `\r` next to `\n`?